databases/aws-rds/manifest.yml

```diff
@@ -7,19 +7,5 @@ applications:
       GOPACKAGENAME: aws-rds
       CGO_CFLAGS: -Isrc/github.com/cloud-gov/databases/aws-rds/include
       LD_LIBRARY_PATH: "/home/vcap/app/include/oracle:$LD_LIBRARY_PATH"
-      SERVICE_NAME: ((pg-service))
-      DB_TYPE: postgresql
-    services:
-      - rds-test-pg
-  - name: aws-rds-smoke-test-suite-oracle
```

Comment:
We still have smoke tests for Oracle. Are we sure that we can remove this?

https://github.com/cloud-gov/aws-broker/blob/main/ci/pipeline.yml#L413

Reply:
Yes, that wasn't being used as it doesn't work. The DB_TYPE is overridden for each test. The Oracle tests are still happening with this change.